### DISCUSSION TITLE
Kibana , logstash services down

### DISCUSSION THREAD

**Discussion by akshayfedee** [telekom-security/tpotce#771] [General]
![1](https://user-images.githubusercontent.com/44234355/108009000-90188d00-6ff9-11eb-99d8-694f29298675.jpg)

hi there, i have been trying to connect the services for elk, but it wont somehow. could you please help on this

**Comment by RogerWeihrauch** [comment]
Hi
How did you try to connect to elk? From external/internal? What did you try til now?
Did you try a restart? Why did you connect via Putty? Which System Hardware is used? ...
Seems some more information may be needed to be able to support you. You really need to be more specific.
